cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -100,6 +100,9 @@ public final class SystemArguments {
 
   public static final String NAMESPACE_CONFIG_PREFIX = "system.namespace.config.";
 
+  // Runtime argument to skip cleanup after pipeline run. Applicable only to jobs running natively in kubernetes.
+  public static final String SYSTEM_JOB_SKIP_CLEANUP = "system.job.skipCleanup";
```

Comment:
Since it is related to the runtime system, I would suggest following the convention to have it `system.runtime.cleanup.disabled` (use noun, not verb). Also, there is no "job" concept inside CDAP.